src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -108,7 +107,14 @@ class WalletService(
                     .getPaymentMethodById(it.paymentMethodId.value.toString())
                     .map { paymentMethod -> paymentMethod to it }
             }
-            .flatMap { (paymentMethod, wallet) ->
+            .flatMap {
```

Comment:
```suggestion
            .flatMap { (paymentMethod, wallet) ->
```
explicitate pair arguments here so that below can be replaced in place of it.first and it.second